### PR TITLE
Fix label issue and bump driver

### PR DESCRIPTION
**Pull request by OskarDamkjaer**
Preview: http://fix_multitstatement_label.surge.sh
Before: 
bild

After:
bild

### CODE REVIEW

File: src/browser/modules/Stream/CypherScriptFrame/CypherScriptFrame.tsx [hunk 1]
```diff
@@ -72,7 +72,7 @@ function CypherScriptFrame({
                       data-testid="multi-statement-list-title"
                       {...titleProps}
                     >
-                      <PointerFrameCommand selectedDb={frames[id].cmd}>
+                      <PointerFrameCommand selectedDb={frames[id].useDb || ''}>
```
Comment (jharris4) [inline src/browser/modules/Stream/CypherScriptFrame/CypherScriptFrame.tsx:75]: Nice find, is the precedent to use empty string vs null/undefined when there is no db?

File: NOTICE.txt [hunk 2]
```diff
@@ -223,7 +223,15 @@ Third-party licenses
 │  ├─ long@4.0.0
 │  │  ├─ URL: https://github.com/dcodeIO/long.js.git
 │  │  └─ VendorName: Daniel Wirtz
-│  ├─ neo4j-driver@4.2.2
+│  ├─ neo4j-driver-bolt-connection@4.3.1
```
Comment (jharris4) [inline NOTICE.txt:226]: Nice, licenses update seems to have been missed with the last driver upgrade

**Review by jharris4 — APPROVED** [review]
LGTM

File: src/browser/modules/Stream/CypherScriptFrame/CypherScriptFrame.tsx (hunk 1 above)
Comment (OskarDamkjaer) [reply src/browser/modules/Stream/CypherScriptFrame/CypherScriptFrame.tsx:75]: Fixed this in another commit

**Review by jharris4 — APPROVED** [review]

**Pull request merged**